app/src/app/[lng]/onboarding/setup/page.tsx

```diff
@@ -388,7 +388,9 @@ export default function OnboardingSetup({
     goToNext();
   };
 
-  const { data: cityData } = useGetOCCityDataQuery(data.locode);
+  const { data: cityData } = useGetOCCityDataQuery(data.locode, {
+    skip: data.locode.length ? false : true,
```

Comment:
No need to change this but fun fact - an empty string is false-y in JS. So just data.locode ? ... achieves the same thing.